### DISCUSSION TITLE
Comments from the chair of the WRB WG Prof Cezary Kabala

### DISCUSSION THREAD

**Discussion by StephanMantelISRIC** [iuss-wrb/wrb#34] [General]
Dear colleagues,
I am very happy with your extensive letters and remarks. The development of WRB is critically dependent on the opinions of its users.
As you know, the most important platform for discussion on WRB is the every-year field workshops arranged by the WRB Working Group (this year in Catalunia, soon in September). Around 40-50 people from various countries spend a week discussing the issues of soil development and transformation in environmental and anthropogenic contexts and testify the WRB rules and detailed solutions (criteria). Moreover, members of the Working Group try to testify and discuss WRB at each other occasion – during the national or international conferences, workshops, and courses. And finally, we use WRB just for teaching, naming, and mapping soils. The results of the latter are commonly published in journals and reports. As you may expect, these are plenty of remarks and suggestions.
Of course, people say that WRB must be continuously developed in line with new soil findings and the development of new concepts, previously unknown, thus not reflected in the classification system. In general, people agree with this statement. But if we start a discussion on particular solutions, we may discover huge differences in the expectations. The best example are Technosols. It is clear that these soils existed always (at least since the beginning of modern soil science history) and people agreed to introduce respective units. But you know how long it took to introduce Technosols as a reference group and how many people were opposed to this. Many national classifications still ignore these soils. Soil Taxonomy probably will introduce a similar order, but many ‘conservative’ soil scientists in the USA do not agree with this ‘unnecessary’ revolution. We always discuss how people will welcome a new unit (at any level) and introduce it when the requests and argumentation are very strong. Typically, proposals for new diagnostic horizons or materials have been discussed for many years. This was the case of finally introduced pretic, murshic, and mulmic. Whereas, the international importance of brunic, a sandy ‘brother’ of the cambic horizon, recognised in some national classifications for many years, is still questioned. In general, most of the proposals are derived from existing national classifications. 

So, we develop the WRB, because of general expectation, new finding and concepts, and strong pressure to involve some national/local solutions, which may have regional/global importance. It is always connected with the basic question you asked: What is WRB? A general platform for correlation only or the comprehensive international system for soil classification? I feel that supporters of the first option are now in a decreasing minority. With the introduction of Technosols, WRB started a transformation towards the second option. 

The development by adding new solutions always results in a growing complication of the system. This is the source of discussion and friction in the Working Group. People consider some solutions to be absolutely necessary and ignore the others. But other people have exactly the opposite opinion. Who is right? Finally, each new solution in the WRB is the result of compromise. You may say that compromise is good in politics, not in science, but the coordination of the global soil classification already is a politics…

When changing the criteria/definitions or introducing new units, we follow proposals of their authors. We try to check/test it, but sometimes the problems are discovered after some time. The best examples are criteria for pretic and chernic in 2014/2015. As early as in 2017 we were sure, the criteria lead to an overestimation of Pretic Technosols and underestimation of Chernozems (while overestimation of Kastanozems). But in both cases, the criteria were constructed based on the proposals of highly experienced and famous professors, supported by many other renowned soil scientists… We really tried to simplify some solutions, but the request of some people and societies to introduce the ‘consequent’ or ‘systematic’ solutions always resulted in their complication (the best case: epic, endic, dorsic). And one more: traditions. This is the crucial problem when we try to change methods/criteria for base saturation and respective horizons/properties. We know the problem you highlighted (‘Eutric Umbric’) and we must find solution for it, but it requires more data and global analysis (eg, if we want to replace complicated and expensive base saturation with simple/cheap pH).  
This is why the newly introduced concepts must be extensively tested and may be removed or modified in the next edition of WRB. Thus, your findings and suggestions are important and will be considered - but not now. The next edition is not planned earlier as in 2030. I am sure that the ‘overcomplication’ at various levels (criteria, qualifiers, rules) will be one of the most common topics discussed in the future. If the pressure from the societies for simplification will be strong, we will remove some solutions and will be more assertive in case of new demands... But we cannot stop a development at all.

The new guidelines for soil description were necessary. Since the introduction of its previous versions (as FAO guidelines), several quantitative criteria in the WRB have changed, and the limits for particular classes in the guidelines disagreed with those in a classification. Moreover, many colleagues request for an international standard for soil horizon designation, at least for publishing in international journals and project reports (we cannot accept national symbols, e.g. Russian or Chinese, in Geoderma or Catena, because they differ drastically from those commonly used in Western Europe or USA, and thus are unclear and confusing for many researchers). Of course, when we started the discussion of the guidelines, plenty of new proposals have been submitted from societies, in particular to add new horizon designations. The largest problems we had were with the definition of the B horizon. We found huge differences in understanding/concepts of this horizon in the national classifications and opinions of colleagues. Similarly, the pedological sense of stagnic/gleyic phenomena and properties generated a huge discussion. I am sure that the discussion will escalate as people realise the new solution in Annex 1. I hope it will generate the critical analysis of the B-horizon concept published in any of international journals… For this edition, we decided to involve the guidelines for soil description in one book. But I think future WRB editions will be published separately from the field guidelines.

Once again, thank you for your valuable comments, I hope not the last!      
  
Warm greetings, Cezary Kabala

**Comment by cyrus621** [comment]
Dear colleagues, sorry that I am late to find out about this discussion forum. Responding here to Cezary: "What is WRB? A general platform for correlation only or the comprehensive international system for soil classification?" I would like to add "A naming system for map units at various scales".  This of course was the explicit aim of the WRB predecessor Legend for the Soil Map of the World which was not a classification, rather legend names, with then descriptions of what the map user might find in the mapped areas of each legend category. 

So from the beginning the RSG+1 qualifier were a continuation of the Legend and could in fact be used directly to re-name the units on the SMW. 

Then, when cross-border projects needed a common legend, the WRB was the obvious choice. Many of you (!!) will recognize this early example: Joisten, H., Jahn, R., Sinaplus, R., Kabala, C., Tomasek, M., Hoffman, R., 2002. The first international joint soil map of Germany, the Czech Republic and Poland in a scale of 1:50000, Sheet Zittau, in: 17th World Congress of Soil Science. International Union of Soil Sciences, Bangkok, p. CD-ROM, paper 2049.

This has implications for the structure of the system. The comparison is to Soil Taxonomy which explicitly aimed its categorical levels at cartographic levels. The Orders could be used for coherent map units of the entire USA, the Suborders then dividing by major climate regions (which is a reason Guy Smith decided to use climate as a criterion), etc. One of the tests of changes would then be how they affect maps at the appropriate scale.

I hope this opens up some discussion. As a geographer first and soil classifier a distant second, I am uncomfortable with the emphasis on allocating pedons to a classification system, just to give a descriptive name which summarizes their properties, without considering the mapping problem. Which after all is how the soils will be identified for land uses and models.

**Reply by cyrus621** [reply]
I can add: I think one reason in Soil Taxonomy that there are no Gleysols, but the Aquic moisture regime at the 2nd level (Aqults, Aquolls etc.) is that at the suborder level they are geographically-associated, so a map of Mollisols does not have to be interrupted or described as an association with a hypothetic "Aquisolls" at the Order level, so the National level map is coherent. Guy Smith in the Interviews says that since the Aquic and Udic (e.g.) suborder behave similarly under drainage and occur together, they should be in the same Order.  Then, at the suborder level (a general county-level map for example) they can be distinguished. 

(He also said - p. 66 of the Interviews - "In respect to classification of wet soils, the Europeans are mostly stuck with their former prejudices about it. They want one order for all the wet soils.")

As an example, in North Carolina (USA) I mapped a drainage sequence of  Typic Hapludults, Aquic Hapludults, Aeric Endoaquults, Typic Endoaquluts.  The latter two are WRB Gleysols, the first two are WRB Acrisols. These all have the same parent material and occur together in regular sequence. At the detailed map level they are all shown, in a county-level map the Udults and Aqults can be separated, at a regional map all are shown as Ultisols.
